### ISSUE TITLE
Slice 4: Behavior training on HiPE1 — 5 VideoMAE configs

### ISSUE THREAD

**Issue by SakifKhan98** [SakifKhan98/cattle-vision-framework#4]
## What to build

Train all 5 VideoMAE behavior classification configs on the new RF-DETR-Seg tubelets. This must run on HiPE1 (Tesla V100 16 GB, Docker) due to VRAM requirements and multi-day training duration.

The 5 v2 YAML config files are already created in \`configs/behavior/\` — they are identical to their v1 counterparts except \`labels_csv\` points to \`data/processed/tubelets_rfdetr/labels.csv\` and \`output_dir\` points to \`runs/behavior/videomae_*_v2\`. \`train.py\` only accepts \`--config\`; there is no CLI override for individual YAML fields.

Training configs (same 5 splits as Phase 6, new tubelet data):
1. \`configs/behavior/videomae_cbvd5_v2.yaml\` — train CBVD-5, val CBVD-5 (in-domain)
2. \`configs/behavior/videomae_cvb_v2.yaml\` — train CVB, val CVB (7 classes, in-domain)
3. \`configs/behavior/videomae_combined_v2.yaml\` — train CBVD-5+CVB, val CBVD-5+CVB (7 classes)
4. \`configs/behavior/videomae_cbvd5_to_cvb_v2.yaml\` — train CBVD-5, val CVB (5 classes, cross-domain)
5. \`configs/behavior/videomae_cvb_to_cbvd5_v2.yaml\` — train CVB, val CBVD-5 (5 classes, cross-domain)

Frozen hyperparameters: batch 8, grad accum 4 (effective 32), 30 epochs, lr 5e-5, BF16 mixed precision, gradient checkpointing, early stopping patience 8. Class weights computed from new tubelet label distribution.

HiPE1 training command (per config, in tmux):
```bash
docker run --rm --gpus all --shm-size=16g \
  -v $(pwd)/data:/workspace/data:ro \
  -v $(pwd)/runs:/workspace/runs \
  -v $(pwd)/configs:/workspace/configs:ro \
  -v $(pwd)/src:/workspace/src:ro \
  cattle-behavior \
  python src/behavior/train.py --config configs/behavior/videomae_combined_v2.yaml
```

See \`docs/design/phase3b_pipeline_rerun_prd.md\` §9.2 for full sync + training + retrieval commands.

## Acceptance criteria

- [ ] All 5 configs complete training without OOM errors
- [ ] Checkpoints saved to \`runs/behavior/videomae_*_v2/checkpoint_best.pt\`
- [ ] Training logs saved for each config
- [ ] Results retrieved from HiPE1 to local machine via rsync

## Blocked by

- #2 — needs CBVD-5 tubelets
- #3 — needs CVB tubelets
